sim/midas/src/main/cc/Makefile

```diff
@@ -20,7 +20,9 @@ OUT_DIR ?= $(GEN_DIR)
 CLOCK_PERIOD ?= 1.0
 
 override CXXFLAGS += -Wall -I$(midas_dir)
+#-I/opt/xilinx/xrt/include -I/opt/xilinx/xrt/experimental/include -I/home/Xilinx/Vivado/2022.1/tps/boost_1_72_0 -I/home/novo/Work/firesim/sim/firesim-lib/src/main/cc -I/home/novo/Work/firesim/target-design/chipyard/generators/testchipip/src/main/resources/testchipip/csrc -I/home/novo/Work/firesim/target-design/chipyard/toolchains/riscv-tools/riscv-isa-sim -I/home/novo/Work/firesim/target-design/chipyard/toolchains/riscv-tools/riscv-isa-sim/build/
```

Comment:
You shouldn't have to change this file, but instead you might have to change some of the vitis-specific arguments in `target-agnostic.mk` depending on your system

Reply:
See these two lines 
https://github.com/firesim/firesim/blob/3647177cebfa0e0fc693a940242f498c7c95d3cc/sim/target-agnostic.mk#L191